src/client/util/http.cc

```diff
@@ -338,54 +340,73 @@ bool HTTP::ProcessI2PResponse() {
+          auto colon = header.find(':');
+          if (colon != std::string::npos)
+            {
+              std::string field = header.substr(0, colon);
+              header.resize(header.length() - 1);  // delete \r
+              // We currently don't differentiate between strong or weak ETags
+              // We currently only care if an ETag is present
+              if (field == "ETag")
+                SetETag(header.substr(colon + 1));
+              else if (field == "Last-Modified")
+                SetLastModified(header.substr(colon + 1));
+              else if (field == "Transfer-Encoding")
+                is_chunked =
+                    !header.compare(colon + 1, std::string::npos, "chunked");
+            }
+        }
+      // Get content after header
+      std::stringstream content;
+      while (std::getline(m_Response, header))
+        {
+          // TODO(anonimal): this can be improved but since we
+          // won't need this after the refactor, it 'works' for now
+          auto colon = header.find(':');
+          if (colon != std::string::npos)
+            continue;
+          else
+            content << header << std::endl;
+        }
+      // Test if response is chunked / save downloaded contents
+      if (!content.eof())
+        {
+          if (is_chunked)
+            {
+              std::stringstream merged;
+              MergeI2PChunkedResponse(content, merged);
+              SetDownloadedContents(merged.str());
+            }
+          else
+            {
+              SetDownloadedContents(content.str());
+            }
+        }
     }
-    // Get content after header
-    std::stringstream content;
-    while (std::getline(m_Response, header)) {
-      // TODO(anonimal): this can be improved but since we
-      // won't need this after the refactor, it 'works' for now
-      auto colon = header.find(':');
-      if (colon != std::string::npos)
-        continue;
-      else
-        content << header << std::endl;
+  else if (
+      response_code == http::basic_response<http::tags::http_server>::
+                           status_type::not_modified)
+    {
+      LOG(info) << "HTTP: no new updates available from " << GetURI().host();
     }
-    // Test if response is chunked / save downloaded contents
-    if (!content.eof()) {
-      if (is_chunked) {
-        std::stringstream merged;
-        MergeI2PChunkedResponse(content, merged);
-        SetDownloadedContents(merged.str());
-      } else {
-        SetDownloadedContents(content.str());
-      }
+  else
+    {
+      LOG(warning) << "HTTP: response code: " << response_code;
+      return false;
     }
-  } else if (response_code
-             == static_cast<std::uint16_t>(ResponseCode::HTTP_NOT_MODIFIED)) {
-    LOG(info) << "HTTP: no new updates available from " << GetURI().host();
-  } else {
-    LOG(warning) << "HTTP: response code: " << response_code;
-    return false;
-  }
```

Comment:
@guzzijones why on earth did you run clang-format on the entire function? When did I ever ask you to do this? The guide clearly says to not mix formatting with non-formatting commits (unless there is heavy refactor work / related refactor work in the process).

This is supposed to be a simple PR: a simple 4 line swap-out for enumerators, 2 additional lines (header and namespace directive), and 6 subtracted lines (removing scoped enum). Can you please save us both the time and keep this PR simple?